assets/src/edit-story/components/helpCenter/useHelpCenter.js

```diff
@@ -102,6 +136,28 @@ export function useHelpCenter() {
     [store.actions]
   );
 
+  useEffect(() => {
+    let isMounted = true;
+    getCurrentUser()
+      .then(({ meta }) => {
+        isMounted &&
+          actions.hydrateReadTipsSuccess({
+            readTips: meta.web_stories_onboarding ?? {},
+          });
+      })
+      .catch(actions.persitingReadTipsError);
+    return () => (isMounted = false);
+  }, [actions, getCurrentUser]);
+
+  useEffect(() => {
+    // don't request to persist tips that are already read
+    Object.keys(prevTips ?? {}).length !==
+      Object.keys(store.state.readTips).length &&
+      updateCurrentUser({
+        meta: { web_stories_onboarding: store.state.readTips },
+      }).catch(actions.persitingReadTipsError);
+  }, [store.state.readTips, actions, prevTips, updateCurrentUser]);
```

Comment:
So this is only persist the tips if they've actually updated?

Reply:
You certainly don't have to do this, but if we wanna get rid of the `usePrev` hook we could do something like this:
```
  // Statically defined
  const createKeyFromBooleanMap = (map) => Object.keys(map || {}).join(' ');
  const createBooleanMapFromKey = (key) => key
      .split(' ')
      .reduce((accum, keys) => {
        accum[key] = true;
      }, {});

  // later in component
  const persitanceKey = createKeyFromBooleanMap(store.state.readTips);
  useEffect(() => {
      updateCurrentUser({
        meta: { web_stories_onboarding: createBooleanMapFromKey(persitanceKey) },
      }).catch(actions.persitingReadTipsError);
  }, [actions, updateCurrentUser, updateKey]);
```

Reply:
let's do it